.eslintrc.json

```diff
@@ -11,7 +11,7 @@
   ],
   "rules": {
     "indent": ["error", 4],
-    "linebreak-style": ["error", "unix"],
```

Comment:
breaks on windows runner... we could switch to warn?

Reply:
> D:\a\playpass\playpass\jest.config.js
> Error:    1:30  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

I don't really understand why this is breaking, aren't our source files using unix line feeds? Do they get transformed when the repo is cloned on Windows?

Anyways, let's just delete the eslint rule for now.